staging/src/k8s.io/client-go/tools/cache/delta_fifo.go

```diff
@@ -99,7 +99,7 @@ type DeltaFIFOOptions struct {
 	EmitDeltaTypeReplaced bool
 }
 
-// NewDeltaFIFOWithOptions returns a Store which can be used process changes to
+// NewDeltaFIFOWithOptions returns a Queue which can be used to process changes to
```

Comment:
hm I missed this on my first pass but I think it actually does implement the Store interface?